<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThresholdToolInterface</class>
 <widget class="QWidget" name="ThresholdToolInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>396</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>213</width>
    <height>396</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>213</width>
    <height>396</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>213</width>
    <height>396</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>207</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="activeHLayout">
    <item>
     <widget class="QLabel" name="activeLabel">
      <property name="text">
       <string>Active:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="activeComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>121</y>
     <width>205</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="lowerHLayout_bottom"/>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>100</y>
     <width>205</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="lowerHLayout_top">
    <item>
     <widget class="QLabel" name="lowerLabel">
      <property name="text">
       <string>Lower Threshold:</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="lowerHSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>205</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="upperHLayout_top">
    <item>
     <widget class="QLabel" name="upperLabel">
      <property name="text">
       <string>Upper Threshold:</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="upperHSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>50</y>
     <width>205</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="upperHLayout_bottom"/>
  </widget>
  <widget class="QPushButton" name="createThresholdLayerButton">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>176</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Threshold Layer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearSeedsButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>208</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Seeds</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="histogramView">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>240</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tempHistogramLabel">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>278</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Histogram goes here.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>370</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="minMaxHLayout">
    <item>
     <widget class="QLabel" name="minLabel">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>min</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="maxLabel">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>max</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
